src/extension.js

```diff
@@ -159,7 +159,7 @@ const ServiceToggle = GObject.registerClass({
                 '%d Connected',
                 '%d Connected',
                 available.length
-            ).format(available.length);
+            ).replace('%d', available.length);
```

Comment:
I'm not the biggest fan of this. It's sort of what GJS does anyways, but I think we just suck it up and do `String.prototype.format = imports.format.format` in `setup.js` like GNOME Shell  does.